modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -64,7 +64,7 @@ metadata:
 ----
 $ oc apply -f _<ns.yaml>_ <1>
 ----
-<1> Replace _<ns.yaml>_ with the name of your namespace configuration file. In this example, `olm-microshift` is used.
+<1> Replace `<ns.yaml>` with the name of your namespace configuration file. In this example, `olm-microshift` is used.
```

Comment:
Remove the italics highlighting in the code example: 

![Screenshot from 2024-09-02 16-03-47](https://github.com/user-attachments/assets/d3493262-c2c8-4ac7-8247-8091f6e35e1f)

Reply:
No, Microshift follows using italics in command. Please refer https://github.com/openshift/openshift-docs/pull/78516#discussion_r1672436701 for the discussion on the itaclics used in Microshift.
Lines 65 and 67 are modified to match the 95,97 and 136,134 in the same file as suggested below